src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static Map<Fruit, Integer> fruits = new HashMap<>();
+
+    public static Map<Fruit, Integer> getFruits() {
+        return fruits;
+    }
```

Comment:
do we need this method if our `fruits` is public?
or we can make the field private